kale/utils/initialize_nn.py

```diff
@@ -0,0 +1,21 @@
+import torch.nn as nn
```

Comment:
doc at top

Reply:
I have added a docstring at the top of the file